### ISSUE TITLE
Quantile Regression in Keras

### ISSUE THREAD

**Issue by sachinruk** [fchollet/keras#4125]
Hi all,

Just wondering if it's possible to create a quantile regression loss function in Keras. I've created it using theano below. I created the lambda function because the loss functions in Keras can only accept two inputs. 

```

def tilted_loss(q,y,f):
    e = (y-f)
    return (q*tt.sum(e)-tt.sum(e[(e<0).nonzero()]))/e.shape[0]

qs = [0.01,0.05,0.5,0.95,0.99]
q_loss_f = [lambda y,f,q=q: tilted_loss(q,y,f) for q in qs]
```

If someone can point out how to do something similar using the "K." notation (as shown below) I'm happy to create a pull request. 

```
def mean_absolute_error(y_true, y_pred):
    return K.mean(K.abs(y_pred - y_true), axis=-1)
```

source: https://github.com/fchollet/keras/blob/master/keras/objectives.py
